website/src/content/docs/docs/emitters/clients/advanced-configurations.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Advanced Client Emitter Configurations
+---
+
+import { Aside } from "@astrojs/starlight/components";
+
+## Advanced Configurations for Client Emitters
+
+This guide provides detailed information on advanced configurations available for client emitters in TypeSpec. These configurations allow you to customize the behavior and output of the generated clients for different programming languages.
+
+### Configuration Flags
+
+| **Configuration Name**       | **Type** | **Possible Values** | **C#**            | **Java**          | **Python**        | **JS**            |
+| ---------------------------- | -------- | ------------------- | ----------------- | ----------------- | ----------------- | ----------------- |
+| generate-protocol-methods    | boolean  | true, false         | User configurable | User configurable | Not used          | Not used          |
+| generate-convenience-methods | boolean  | true, false         | User configurable | User configurable | Not used          | Not used          |
+| flatten-union-as-enum        | boolean  | true, false         | Not used          | Not used          | Not used          | Fixed value       |
+| examples-dir                 | string   | directory path      | User configurable | User configurable | User configurable | User configurable |
+| namespace                    | string   | namespace path      | User configurable | User configurable | User configurable | User configurable |
+| api-version                  | string   | version string      | User configurable | User configurable | User configurable | User configurable |
```

Comment:
@ joheredi I see a lot of options the js emitters is says is configurable but right noiw only `pacakge-name` is defined